finance/beancount/Portfile

```diff
@@ -1,35 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           python 1.0
-PortGroup           bitbucket 1.0
-
-bitbucket.setup     blais beancount bd3f97caca3d
+PortGroup           obsolete 1.0
 
 name                beancount
+replaced_by         py-beancount
 categories          finance
 license             GPL-2
 version             0.1
```

Comment:
Does this port need a revision bump now?

Reply:
Seems so; bumped.